TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.h

```diff
@@ -101,6 +101,8 @@ class Moveable : public NamedBase<Moveable, short>
 	void MakeInvisible();
 	void Explode();
 	void Shatter();
+	void SetPoison(int $potency);
```

Comment:
Remove the `$` in front of the arg name, as it will probably confuse people into thinking it has some special behaviour in C++ (I didn't even think it was legal to start an identifier with a dollar sign)